django/db/models/functions/json.py

```diff
@@ -0,0 +1,23 @@
+from django.db.models.expressions import Func, Value
+from django.db.models.fields.json import JSONField
+
+
+class JSONObject(Func):
+    function = 'JSON_OBJECT'
+    output_field = JSONField()
+
+    def __init__(self, **fields):
+        expressions = []
+        for key, value in fields.items():
+            expressions.extend((Value(key), value))
```

Comment:
```suggestion
        expressions = [
            (Value(key), value) for key, value in fields.items():
        ]
```

Reply:
It will be list of tuples. Not flat list.

Reply:
@Chiorufarewerin ahh you're right I missed that.